projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -45,7 +45,7 @@ public static function init() {
 		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'unbind_focusout_on_wp_admin_bar_menu_toggle' ) );
 
 		// Load the Map block settings.
-		add_action( 'enqueue_block_assets', array( __CLASS__, 'load_map_block_settings' ) );
+		add_action( 'enqueue_block_assets', array( __CLASS__, 'load_map_block_settings', 999 ) );
```

Comment:
I think you'd have to move the priority outside of the array here:

```suggestion
		add_action( 'enqueue_block_assets', array( __CLASS__, 'load_map_block_settings' ), 999 );
```

Reply:
🫠 My bad. I first (wrongly) made the PR on the wrong repo. I then used the Github editor to quickly make the fix here. I wanted to go too quickly. Thanks for catching this.